includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -680,6 +680,11 @@ public function get_oauth_data( $return_url, array $business_data = [], array $s
 				'site_data'           => $site_data,
 				'create_live_account' => ! WC_Payments::get_gateway()->is_in_dev_mode(),
 				'actioned_notes'      => $actioned_notes,
+				'client_data'         => [
+					//phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+					'user_agent' => $_SERVER['HTTP_USER_AGENT'] ?? '',
```

Comment:
Wouldn't it be better to sanitize this input in case someone sends a malicious user-agent?

Reply:
Removed in https://github.com/Automattic/woocommerce-payments/pull/1349/commits/d2be9becfe7136628eaa52143be9d40b081495ef